MEIOUandTaxes1/src/events/DLC-MamlukEvents.txt

```diff
@@ -661,3 +661,54 @@ country_event = {
 		}
 	}
 }
+
+##Starting situation in Mamluk court
+country_event = {
+	id = mamluk_events.10
+	title = mamluk_events.10.t
+	desc = mamluk_events.10.d
+	picture = MAMLUK_WARRIORS_eventPicture
+
+	trigger = {
+		tag = MAM
+	}
+	
+	fire_only_once = yes
+	
+	mean_time_to_happen = {
+		months = 1
+	}
+
+	option = {
```

Comment:
This option is straight worse than the normal demote nobles interaction, and its alternative is to do nothing. There's no incentive for the player to ever do this. Would like to see it changed to either include a beneficial effect here (maybe additional noble power reduction, or some other modifier?) or add a detrimental effect to the other option.